pkg/node/node.go

```diff
@@ -93,32 +93,6 @@ func NewNodeServer(ctx context.Context, identity, nodeID, rack, zone, region str
 		setQuota:        xfs.SetQuota,
 	}
 
-	if dynamicDriveDiscovery {
-		handler := &ueventHandler{
-			nodeID: nodeID,
-			topology: map[string]string{
-				string(utils.TopologyDriverIdentity): identity,
-				string(utils.TopologyDriverRack):     rack,
-				string(utils.TopologyDriverZone):     zone,
-				string(utils.TopologyDriverRegion):   region,
-				string(utils.TopologyDriverNode):     nodeID,
-			},
-			dynamicDriveDiscovery: dynamicDriveDiscovery,
-			loopbackOnly:          loopbackOnly,
-		}
-		if loopbackOnly {
-			if err := sys.CreateLoopDevices(); err != nil {
-				return nil, err
-			}
-		}
-
-		klog.V(3).Info("Doing initial drive sync up")
-		handler.syncDrives(ctx)
-
-		// Start background tasks
-		go handler.processLoop(ctx)
-	}
```

Comment:
if we decide to support backward compatibility, we need to keep this code. we need to make this code segment run, if `--dynamic-drive-discovery` flag is set.

Reply:
It is going to run as a separate container. This is not needed.